### DISCUSSION TITLE
Show last 15 test runs in Scenario Trend

### DISCUSSION THREAD

**Discussion by gelsalgado** [ludeknovy/jtl-reporter#202] [Q&A]
Hello! I know that the scenario trend only shows the first 15 test runs and some users have resorted to deleting old tests so they can show the latest results in the scenario trend.

Is there a way so that I can show the last 15 test runs instead without deleting my old performance tests?

**Comment by ludeknovy** [comment]
Hi @gelsalgado ! the scenario trend chart should display the latest 15 test runs. You definitely are not supposed to delete test runs to display them the way you want.

What version are you using?

**Reply by ludeknovy** [reply]
The above-mentioned issue was resolved here: https://github.com/ludeknovy/jtl-reporter/issues/193

**Reply by gelsalgado** [reply]
Hi! Thanks for the reply. I am currently using 4.7.1. How can I check for certain that my JTL reporter is running the correct version? 

As you can see, the last data in the scenario trend is not the last set of test I executed.
![image](https://user-images.githubusercontent.com/28947736/220517191-b3206904-609c-4201-adc0-0e54b9a47c6d.png)

**Reply by gelsalgado** [reply]
These are the containers running in my machine.
![image](https://user-images.githubusercontent.com/28947736/220521978-faaf02e8-df5e-4da0-8293-4125403ac71a.png)

**Reply by ludeknovy** [reply]
Hi @gelsalgado! The screenshot you provided reads v4.4.3 of the jtl-reporter-be. The fix of the issue above was released in v4.5.0 - see https://github.com/ludeknovy/jtl-reporter/releases/tag/v4.5.0.

Please try to update the application to v4.7.1

**Reply by gelsalgado** [reply]
Hi @ludeknovy! Thank you! After you pointed it out, I figured I had to stop my containers and start them again. Now I am running and it works like a charm! Thanks again!

**Reply by ludeknovy** [reply]
Perfect, glad you solved it..
